src/app/space/create/apps/apps.component.html

```diff
@@ -0,0 +1,99 @@
+<div id="apps" class="">
+
+  <div class="cards-pf">
+    <div class="row row-cards-pf">
+      <div class="col-xs-12 col-sm-4 col-md-2"><h3><strong>Applications</strong></h3></div>
```

Comment:
Can you break these into separate lines for readability?
```
<div>
   <h3>
    <strong>Applications</strong>
  </h3>
</div>
```

Also, is there a reason why `<strong>` rather than `<b>`? If this is just to add emphasis, then `<b>` should be used. `<strong>` is only used to provide added importance to an object.